stable/acs-engine-autoscaler/values.yaml

```diff
@@ -23,4 +23,8 @@ acsenginecluster:
   #ignorepools:
   ## Optional parameter denominating number of nodes to spare in a pool after scaling
   #spareagents:
+  ## Optional parameter denominating the maximum duration (in seconds) an agent can stay idle before being deleted
+  #idle-threshold:
+  ## Optional parameter denominating the number of extra agents to create when scaling out
+  #over-provision:
```

Comment:
Same here. The values should be `idlethreshold` and `overprovision` as you have provided in the yaml file.